runtime/src/accounts_background_service.rs

```diff
@@ -122,8 +122,11 @@ impl SnapshotRequestHandler {
 
                 let mut hash_time = Measure::start("hash_time");
                 let mut hash_for_testing = None;
-                snapshot_root_bank
-                    .update_accounts_hash_with_index_option(true, test_hash_calculation);
+                const DO_NOT_USE_INDEX: bool = false;
```

Comment:
Fwiw, I find the double-negative a little difficult to follow. If the churn isn't too awful, could we update this to `use_index` in `update_accounts_hash_with_index_option` and etc?